spec/controllers/lectures_controller_spec.rb

```diff
@@ -104,13 +123,12 @@
     end
 
     context "with invalid params" do
-      it "redirects for other lecturers" do
+      it "redirects for other lecturers", :logged_lecturer do
         lecture = Lecture.create! valid_attributes_with_lecturer
-        login_lecturer()
         put :update, params: { id: lecture.to_param, lecture: invalid_attributes }, session: valid_session
         expect(response).to redirect_to(lectures_url)
       end
-      it "returns a success response for lecturer (i.e. to display the 'edit' template)" do
+      it "returns a success response for lecturer (i.e. to display the 'edit' template)", :logged_lecturer do
         lecture = Lecture.create! valid_attributes_with_lecturer
         login_lecturer(lecture.lecturer)
```

Comment:
I am not sure, but is this still needed? The tag `:logged_lecturer` should already log in an lecturer. Here it looks / reads like we are logging the lecturer in again 🤔

Reply:
I mean the `login_lecturer(lecture.lecturer)` in line 115/133

Reply:
Yes it is because it's a special lecturer